autoware_launch/config/system/pipeline_latency_monitor/pipeline_latency_monitor.param.yaml

```
@@ -0,0 +1,57 @@
+/**:
+  ros__parameters:
+    update_rate: 10.0
+    latency_threshold_ms: 1000.0
+    window_size: 10
+
+    processing_steps:  # processing steps used to calculate the latency.
+      # The name of the steps are only used for setting the parameters and for debug outputs
+      sequence:  # sequence ordered from first to last step
+        - map_based_prediction
+        - multi_object_tracker
+        - decorative_object_merger_node
```

Could you rearrange the order to match the actual order of the nodes?
The same goes for the detailed descriptions of each step described below.
```suggestion
        - multi_object_tracker
        - decorative_object_merger_node
        - map_based_prediction
```